### ISSUE TITLE
FrenchSouthWestProvider: NPE with parseColor when color is null

### ISSUE THREAD

**Issue by grote** [schildbach/public-transport-enabler#94]
There's still a `null` color issue:

```
05-27 15:19:39.875 W/System.err: java.lang.NullPointerException
05-27 15:19:39.879 W/System.err:     at com.google.common.base.Preconditions.checkNotNull(Preconditions.java:210)
05-27 15:19:39.887 W/System.err:     at de.schildbach.pte.dto.Style.parseColor(Style.java:116)
05-27 15:19:39.887 W/System.err:     at de.schildbach.pte.FrenchSouthWestProvider.getLineStyle(FrenchSouthWestProvider.java:63)
05-27 15:19:39.887 W/System.err:     at de.schildbach.pte.AbstractNavitiaProvider.parseLineFromSection(AbstractNavitiaProvider.java:494)
05-27 15:19:39.887 W/System.err:     at de.schildbach.pte.AbstractNavitiaProvider.parseLeg(AbstractNavitiaProvider.java:565)
05-27 15:19:39.887 W/System.err:     at de.schildbach.pte.AbstractNavitiaProvider.parseQueryTripsResult(AbstractNavitiaProvider.java:658)
05-27 15:19:39.887 W/System.err:     at de.schildbach.pte.AbstractNavitiaProvider.queryTrips(AbstractNavitiaProvider.java:1142)
```

**Comment by grote** [comment]
Same problem for `FranceSouthEastProvider`. See grote/Transportr#360

@kalon33 and @thony8 you added those providers, so you are the current maintainers. Do you need help fixing these issues?

**Comment by thony8** [comment]
I answered in Transportr repository (https://github.com/grote/Transportr/issues/360) with a pull request (#169)  some GTFS used by Navitia do not use hex color (for example the SNCF, the national train company). I added a default color for this case (for SE, SW and NE)

**Comment by schildbach** [comment]
Should be fixed via 7f607160ca919bd68628d867f0a80972c7a471ae.
